### ISSUE TITLE
IllegalStateException: Not allowed to start service Intent { act=mozac.feature.media.service.UPDATE_STATE cmp=org.mozill...

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/fenix#4723]
https://sentry.prod.mozaws.net/operations/fenix-nightly/issues/6140895/

```
IllegalStateException: Not allowed to start service Intent { act=mozac.feature.media.service.UPDATE_STATE cmp=org.mozilla.fenix/mozilla.components.feature.media.service.MediaService }: app is in background uid UidRecord{2f0b992 u0a163 LAST bg:+1m14s603ms idle change:cached procs:3 seq(0,0,0)}
    at android.app.ContextImpl.startServiceCommon(ContextImpl.java:1595)
    at android.app.ContextImpl.startService(ContextImpl.java:1550)
    at android.content.ContextWrapper.startService(ContextWrapper.java:664)
    at mozilla.components.feature.media.service.MediaService$Companion.updateState(MediaService.kt:3)
    at mozilla.components.feature.media.MediaFeature.notifyService(MediaFeature.kt:11)
...
(24 additional frame(s) were not displayed)

Not allowed to start service Intent { act=mozac.feature.media.service.UPDATE_STATE cmp=org.mozilla.fenix/mozilla.components.feature.media.service.MediaService }: app is in background uid UidRecord{2f0b992 u0a163 LAST bg:+1m14s603ms idle change:cached procs:3 seq(0,0,0)}
```

**Comment by pocmo** [comment]
This only affects Nightly builds (feature is behind a Nightly flag). Filing to address this in AC.

**Comment by pocmo** [comment]
AC issue: https://github.com/mozilla-mobile/android-components/issues/4121

**Comment by pocmo** [comment]
Fixed in 9.0.0.
